app/javascript/packs/application.js

```diff
@@ -17,7 +17,6 @@ require("../includes/infinitescroll.js")
 require("../includes/navbar.js")
 require("../includes/discussion.js")
 require("../includes/comment_form.js")
-require("../includes/autocomplete.js")
```

Comment:
why did you remove the autocomplete.js? is it your code or someone else's?

Reply:
Auto complete js is not working. Because of that file, some Ajax functions didn't work. So removed that file